Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY ./requirements.txt /app/requirements.txt
 RUN apt-get -q update \
     && apt-get -q --yes install g++ \
     && pip install --upgrade --no-cache-dir -r requirements.txt \
-         gunicorn nose flake8 \
+         gunicorn nose flake8 mock webtest unittest2 \
```

Comment:
+1 to use dev-requirements.txt now that it exists.
Apparently they want to run tests from the Docker and it is the only place to install the dependencies in it.